packages/url.lua

```diff
@@ -40,8 +40,11 @@ SILE.registerCommand("url", function (options, content)
   SILE.call("code", {}, result)
 end)
 
-SILE.registerCommand("code", function(options, content)
-  SILE.call("verbatim:font", options, content)
+SILE.registerCommand("code", function(_, content)
+  SILE.settings.temporarily(function()
```

Comment:
Good catch.

This is ... indicative of a different problem. The `verbatium:font` command is not processing input. I think I'd rather fix this at the source. Of course this command isn't the only offender that doesn't process content even if sent, but lets weed them out rather than cater to them.

Reply:
True, I was also thinking about modifying `verbatim:font` instead, so doing this now.